src/python/dxpy/nextflow/nextflow_builder.py

```diff
@@ -56,6 +56,15 @@ def build_pipeline_from_repository(repository, tag, profile="", github_creds=Non
         print(applet_id)
     return applet_id
 
+def get_resources_dir_name(resources_dir):
+    """
+    :param resources_dir: Directory with all source files needed to build an applet. Can be an absolute or a relative path.
+    :type resources_dir: str or Path
+    :returns: The name of the folder
+    :rtype: str
+    """
+    return os.path.basename(os.path.abspath(resources_dir))
```

Comment:
Actually I changed it as well because if the resources_dir happens to end with a `/`, i.e. `dx build --nextflow test/nextflow/` it would work still but the name of the applet would be empty.

Reply:
Could you provide an example where the name ends with / and the name end up empty? I don't see such a problem (expect for when it's /)

```
>>> resources_dir = "foo/bar/"
>>> os.path.basename(os.path.abspath(resources_dir))
'bar'
>>> resources_dir = "/"
>>> os.path.basename(os.path.abspath(resources_dir))
''
```

Reply:
```
>>> os.path.basename('test/nextflow/')
''
>>> os.path.basename(os.path.abspath('test/nextflow/'))
'nextflow'
```
if my `main.nf` is stored under test/nextflow

I think your code has already made it fixed. Just want to comment on it and suggest another test. No further change needed. Sorry for the confusion!